test/main.ts

```diff
@@ -17,7 +17,7 @@ const archiver = require('archiver')
 
 test('Downloads the Deno CLI on demand and caches it for subsequent calls', async (t) => {
   const latestVersion = '1.20.3'
-  const mockBinaryOutput = `#!/usr/bin/env node\n\nconsole.log("deno ${latestVersion}")`
+  const mockBinaryOutput = `#!/usr/bin/env sh\n\necho "deno ${latestVersion}"`
```

Comment:
I had to change on test, because the mock for the deno cli was using node, but node couldn't be found anymore, because PATH is not set